hivemind/moe/__init__.py

```diff
@@ -1,5 +1,6 @@
 from hivemind.moe.client import RemoteExpert, RemoteMixtureOfExperts, RemoteSwitchMixtureOfExperts
 from hivemind.moe.server import (
+    ConnectionHandler,
```

Comment:
```suggestion
```

We usually only export "the most public" classes to the root of the `hivemind` namespace (i.e., the classes most useful for a developer that uses hivemind).

Is there a reason to export `ConnectionHandler`? I thought this was an internal class.

Reply:
I think there is no reason. I'll remove it